### ISSUE TITLE
Direct casts

### ISSUE THREAD

**Issue by alexreyes** [standard-crypto/farcaster-js#527]
**Is your feature request related to a problem? Please describe.**
Problem: you can build 90% of a farcaster client only using farcasterjs (which is amazing!) but it's missing direct casts.

**Describe the solution you'd like**
An API for retrieving and sending direct casts

**Describe alternatives you've considered**
Haven't looked into it too much but the alternative is to mess with farcaster hubs

**Comment by gsgalloway** [comment]
Good call, this is indeed missing. I'll need to take a look at the way clients manage direct casts currently.

I don't believe that the Warpcast REST API provides this functionality, or at least publicly -- implementing this then might hinge on pulling in the support for hub-specific functionality that has been on the TODO list for a bit now. I'll post learnings here

**Comment by alexreyes** [comment]
@gsgalloway great! Curious to hear what you find in this thread. It would be super helpful for a alt client I'm working on for fun rn.

**Comment by gsgalloway** [comment]
This is likely dependent on issue #540
